examples/chart/teleport-agent/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+name: teleport-agent
+apiVersion: v2
+version: 0.0.1
+description: Teleport provides a secure SSH and Kubernetes remote access solution that doesn't get in the way.
+icon: https://gravitational.com/gravitational/images/logos/company-logos/teleport-symbol-400x400.png
```

Comment:
We should make this the new logo, I'll try to get it fro the UX crew.

Reply:
thanks, changed